ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -346,6 +353,8 @@ open class SideTabBar: UIView, TokenizedControlInternal {
         for subview in stackView(in: section).arrangedSubviews {
             if let tabBarItemView = subview as? TabBarItemView {
                 let tabBarItemTokenSet = tabBarItemView.tokenSet
+                let badge = tabBarItemView.badgeView as? BadgeLabel
```

Comment:
I think if you do `if let badge = tabBarItemView.badgeView as? BadgeLabel` you can remove the `?` from `badge?.style = badgeStyle`

Reply:
Yep I'll do that. thanks!